utopia-remix/app/routes/online-status.tsx

```diff
@@ -1,3 +1,15 @@
-export async function loader() {
+import type { LoaderFunctionArgs } from '@remix-run/node'
+import { proxy } from '../util/proxy.server'
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  // Call the old server to check if it's online.
+  const url = new URL(request.url)
+  url.pathname = '/online-status'
+  const onlineStatusRequest = new Request(url, {
+    method: 'GET',
+  })
+  await proxy(onlineStatusRequest, { rawOutput: true })
+
+  // Return a simple response.
   return new Response('Online', { headers: { 'content-type': 'text/plain' } })
```

Comment:
I missed this, we should include `cache-control: no-cache` here to please the cloudfront deities

Reply:
(since this is not going through `handle`)